verseek_lib/__init__.py

```diff
@@ -257,28 +266,32 @@ def seek_version(self, version: Optional[str] = None):
 class GitSingle(Git):
     """version seeking class for git repository containing one package"""
 
-    def __init__(self, path: os.PathLike):
+    def __init__(self, path: AnyPath):
         Git.__init__(self, path)
-        self.autoversion = Autoversion(path, precache=True)
+        self.autoversion = Autoversion(os.fspath(path), precache=True)
```

Comment:
Or we should get rid of the `fspath()` function...

Reply:
FWIW I created the fspath function because I incorrectly thought a `Path` object could have a `str` OR `bytes` as it's underlying type, this is not correct and so the function eventually got stripped down to being absolutely useless :P

Reply:
Oh no I'm mistaken, this function was created to use a `str`, `bytes` or `os.PathLike` object interchangeably. I removed bytes but it's still useful where you explicitly want to only support a `str` or `os.PathLike`

Reply:
So, to answer your question, yes it should be an `fspath` rather than `os.fspath` I'll handle that later today if you want

Reply:
No that's cool man. I'll do it and push as I've just noticed that I have mode some other (relatively minor packaging related changes) so I'll open a new PR with those included (and this fix).